### ISSUE TITLE
'ResidualBlock' object has no attribute 'dropout'

### ISSUE THREAD

**Issue by qingweihk** [skumra/robotic-grasping#15]
When  I run evaluate.py   (  --network trained-models/cornell-randsplit-rgbd-grconvnet3-drop1-ch32/epoch_19_iou_0.98   --dataset jacquard  --dataset-path Samples     --jacquard-output --iou-eval)

and got an error:'ResidualBlock' object has no attribute 'dropout'.

Thanks

**Comment by skumra** [comment]
Are you using latest `develop` or `release 0.2.1`?

**Comment by catisthebest** [comment]
hi ~ i have the same problem ,when i run the command below:
python run_offline.py  --network trained-models/cornell-randsplit-rgbd-grconvnet3-drop1-ch32/epoch_19_iou_0.98 

i have try the release 0.2.1 code, and still got the same problem.

my Env is:
ubuntu 18.04
cuda 10.1
pytorch 1.7
python 3.6

could u tell me your env? thanks a lot !  cause i think may be it is a env problem .

**Comment by skumra** [comment]
I was able to reproduce and fix the issue. Please try latest `master` or `release v0.2.2`
